Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeDrawingAreaProxyMac.mm

```diff
@@ -305,7 +305,10 @@ explicit RemoteLayerTreeDisplayLinkClient(WebPageProxyIdentifier pageID)
     NSValue *transformValue = [NSValue valueWithCATransform3D:transform];
     [renderViewAnimationCA setToValue:transformValue];
     
-    [CATransaction setCompletionBlock:[layerForPageScale, this, scale, constrainedOrigin, transform] () {
+    [CATransaction setCompletionBlock:[this, scale, constrainedOrigin, transform] () {
+        CALayer *layerForPageScale = remoteLayerTreeHost().layerForID(m_pageScalingLayerID);
+        if (!layerForPageScale)
+            return;
```

Comment:
We should still set `m_transactionIDAfterEndingTransientZoom` and call `m_webPageProxy.send(Messages::DrawingArea::CommitTransientZoom...` right?

Reply:
Hmm im not sure exactly what would cause this layer to go away. Would it still make sense to commit the zoom in the web process in this case?

Reply:
This may happen on page navigation. Leaving the web process without a committed zoom sounds like a bad idea.

Reply:
Ah ok gotcha. Updated.